Documentation/technical/commit-graph-format.txt

```diff
@@ -93,6 +96,21 @@ CHUNK DATA:
       positions for the parents until reaching a value with the most-significant
       bit on. The other bits correspond to the position of the last parent.
 
+  Bloom Filter Index (ID: {'B', 'I', 'D', 'X'}) (N * 4 bytes) [Optional]
+      The ith entry, BIDX[i], stores the number of 8-byte word blocks in all
+      Bloom filters from commit 0 to commit i (inclusive) in lexicographic
+      order. The Bloom filter for the i-th commit spans from BIDX[i-1] to 
+	  BIDX[i] (plus header length), where BIDX[-1] is 0. 
+
+  Bloom Filter Data (ID: {'B', 'D', 'A', 'T'}) [Optional]
+    * It starts with header consisting of three unsigned 32-bit integers:
+      - version of the hash algorithm being used
```

Comment:
Perhaps be specific that we currently only support the value 1 meaning `murmur3` with initial seeds X and Y?